openrtx/include/protocols/M17/M17Demodulator.hpp

```diff
@@ -115,54 +114,61 @@ class M17Demodulator
      */
     void invertPhase(const bool status);
 
-private:
-
+   private:
     /**
      * M17 baseband signal sampled at 24kHz, half of an M17 frame is processed
      * at each update of the demodulator.
      */
-    static constexpr size_t M17_RX_SAMPLE_RATE     = 24000;
-
-
-    static constexpr size_t  M17_SAMPLES_PER_SYMBOL = M17_RX_SAMPLE_RATE / M17_SYMBOL_RATE;
-    static constexpr size_t  M17_FRAME_SAMPLES      = M17_FRAME_SYMBOLS * M17_SAMPLES_PER_SYMBOL;
-    static constexpr size_t  M17_SAMPLE_BUF_SIZE    = M17_FRAME_SAMPLES / 2;
-    static constexpr size_t  M17_SYNCWORD_SAMPLES   = M17_SAMPLES_PER_SYMBOL * M17_SYNCWORD_SYMBOLS;
-    static constexpr int8_t  SYNC_SWEEP_WIDTH       = 10;
-    static constexpr int8_t  SYNC_SWEEP_OFFSET      = ceil(SYNC_SWEEP_WIDTH / M17_SAMPLES_PER_SYMBOL);
-    static constexpr int16_t M17_BRIDGE_SIZE        = M17_SYNCWORD_SAMPLES + 2 * SYNC_SWEEP_WIDTH;
-
-    static constexpr float  CONV_STATS_ALPHA       = 0.005f;
-    static constexpr float  CONV_THRESHOLD_FACTOR  = 3.40;
-    static constexpr int16_t QNT_SMA_WINDOW        = 8;
+    static constexpr size_t M17_RX_SAMPLE_RATE = 24000;
```

Comment:
Ok that we overflowed the 80 columns, but it was more readable... Probably we have to add lint exceptions here and there